dask/dataframe/core.py

```diff
@@ -96,7 +97,13 @@
 
 DEFAULT_GET = named_schedulers.get("threads", named_schedulers["sync"])
 
-no_default = "__no_default__"
+
+class _NoDefault(Enum):
+    no_default = ...
+
+
+no_default: Final = _NoDefault.no_default
+NoDefault = Literal[_NoDefault.no_default]
```

Comment:
This change was needed to make `mypy` happy with the new default value of `use_nullable_dtypes=` in `read_parquet`. cc @crusaderky in case you think there's a better way to do this

Reply:
This is indeed the canonical way of doing it (and yes it's ugly)